### PR TITLE
implemented fixes to bosh v2 commands 

### PR DESCRIPTION
**Pull request by gwaldo**
implemented fixes to bosh v2 commands per https://help.datadoghq.com/hc/en-us/articles/360002921491

### What does this PR do?
Updates the bosh v2 commands

### Motivation
Jeremy told me to

### Preview link
https://docs.datadoghq.com/integrations/cloud_foundry/#overview

### Additional Notes
This is a first pass at the doc, but I don't understand much about the ecosystem.  Someone more knowledgable with Pivotal Cloud Foundry might be able to get more value from it.

### CODE REVIEW

**Comment by gwaldo** [comment]
@technovangelist Would you have a look at this?

I corrected all of the bosh commands until the final section ("Configure New Deployment for Nozzle Jobs"), who's work doesn't seem to be represented in the existing doc.  I don't know enough about it to say whether it's needed.

File: content/integrations/cloud_foundry.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ Datadog provides tarballs of the Datadog Agent packaged as a BOSH release. Uploa
 bosh upload release https://cloudfoundry.datadoghq.com/datadog-agent/datadog-agent-boshrelease-latest.tgz
 
 # BOSH CLI v2
-bosh upload-release https://cloudfoundry.datadoghq.com/datadog-agent/datadog-agent-boshrelease-latest.tgz
+bosh upload-release -e pcf https://cloudfoundry.datadoghq.com/datadog-agent/datadog-agent-boshrelease-latest.tgz
```
[the remark this answers is not in the document]
Comment (gwaldo) [reply content/integrations/cloud_foundry.md]: Thank you, @arbll.  I'll make that change.

**Comment by gwaldo** [comment]
@arbll @gmmeyer @gzussa Please review.

File: content/integrations/cloud_foundry.md [hunk 2]
```diff
@@ -197,7 +197,19 @@ addons:
 
 To see which `datadog-agent` release version you uploaded earlier, run `bosh releases`.
 
-If you don't have a local copy of the runtime configuration, create a `runtime.yml` with the current runtime configuration (`bosh runtime-config`) and add the above YAML to it. If the Director's runtime configuration is empty, create a new `runtime.yml` containing only the above YAML.
+#### Load the runtime.yml
+
+Check if you have a previously configured runtime-config by running:
```
Comment (phrawzty) [inline content/integrations/cloud_foundry.md]: Is "runtime-config" a command? If so, `backticks`. Is it two words? If so, remove the hyphen.

**Comment by gwaldo** [comment]
updated per @phrawzty 's suggestion

**Comment by gzussa** [comment]
Nice 💯

**Pull request merged**